dist/index.js

```diff
@@ -27,8 +28,37 @@ const octokit = new github.GitHub(myToken);
 
 const AWS_ECR_URL = "public.ecr.aws/cds-snc";
 
```

Comment:
Can we add a constant for the GitHub org? I think we repeat quite a few times the value `cds-snc`